packages/vanilla-components/src/components/copy-button/copy-button.tsx

```diff
@@ -3,24 +3,87 @@ import { shadeColor, addClass, removeClass } from "../../utilities";
 import { css } from "emotion";
 import { API } from "../../services/WidgetHost";
 import Clipboard from "clipboard";
-
+/**
+ * @uiName Copy Button
+ */
 @Component({
   tag: "sqh-copy-button",
   styleUrl: "copy-button.scss",
 })
 export class CopyButton {
+   /**
+   * Hide the component
+   *
+   * @uiName Is Hidden
```

Comment:
Something like Hide Button is more appropriate, this applies to other components as well